### ISSUE TITLE
Release new version?

### ISSUE THREAD

**Issue by sgerrand** [stripe-ruby-mock/stripe-ruby-mock#881]
The most recent version of this gem, [3.1.0.rc3](https://rubygems.org/gems/stripe-ruby-mock/versions/3.1.0.rc3),  was published over 12 months ago and [many changes have subsequently been merged](https://github.com/stripe-ruby-mock/stripe-ruby-mock/compare/master..3.1.0.rc3). Is a new version due to be released soon?

**Comment by sgerrand** [comment]
@TastyPi, what are your thoughts on this topic?

**Comment by stevecondylios** [comment]
I'm new to stripe-ruby-mock but thought the same thing after adding stripe-ruby-mock to Gemfile:

```ruby
group :test do 
  gem 'stripe-ruby-mock', '~> 3.0.1', :require => 'stripe_mock'
end
```

and seeing

```ruby
bundle
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Bundler could not find compatible versions for gem "stripe":
  In snapshot (Gemfile.lock):
    stripe (= 8.6.0)

  In Gemfile:
    stripe (~> 8.6)

    stripe-ruby-mock (~> 3.0.1) was resolved to 3.0.1, which depends on
      stripe (< 6, > 5)

Deleting your Gemfile.lock file and running `bundle install` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Seems the most recent stripe-ruby-mock needs a stripe gem major version between 5-6, but stripe gem is already at major version 8.6.

**Comment by TastyPi** [comment]
Duplicate of #795
